test/extended/util/image/image.go

```diff
@@ -65,7 +65,7 @@ func init() {
 
 		// allowed upstream kube images - index and value must match upstream or
 		// tests will fail (vendor/k8s.io/kubernetes/test/utils/image/manifest.go)
-		"k8s.gcr.io/e2e-test-images/agnhost:2.33":   1,
+		"k8s.gcr.io/e2e-test-images/agnhost:2.39":   1,
```

Comment:
This version is defined by the kubernetes test suite, and is being updated to 2.36 as part of the k8s 1.24.0 rebase in #27181.

Reply:
Also, all the rest of the tests in origin that use `agnhost` also need to be updated to the same version simultaneously.